src/plugins/intel_cpu/src/cpu_streams_calculation.cpp

```diff
@@ -146,7 +146,8 @@ std::vector<std::vector<int>> get_streams_info_table(const int input_streams,
         }
     }
 
-    if (((input_streams_changed == false) && (input_perf_hint == CONFIG_VALUE(LATENCY)) &&
+    if (((input_streams_changed == false) &&
+         (input_perf_hint == ov::util::to_string(ov::hint::PerformanceMode::LATENCY)) &&
```

Comment:
Do we have another way to compare perf_hint? To avoid using "to_string()", maybe we can construct ov::hint::PerformanceMode instance from "input_perf_hint"

Reply:
ov::hint::PerformanceMode is enum class object, I guess you mean to use ov::hint::performance_mode property like this?

`(ov::hint::performance_mode(input_perf_hint).second == ov::hint::PerformanceMode::LATENCY) `

but it will throw exception if `input_perf_hint` is not in {"LATENCY", "THROUGHPUT", "CUMULATIVE_THROUGHPUT"}, so we can consider below code:

`(input_perf_hint == ov::hint::performance_mode(ov::hint::PerformanceMode::LATENCY).second.as<std::string>())`

This code is not too much different with current implement, do you have better idea?